test/test_grid_manipulation.py

```diff
@@ -71,62 +56,41 @@ def test_build_edge_coordinates():
     grid = two_triangles()
     grid.build_edge_coordinates()
     coords = grid.edge_coordinates
-    # nodes = grid.nodes  # FIXME: Assigned but unused!
-    print(coords)
 
     assert coords.shape == (5, 2)
-    assert np.allclose(coords, [[1.1, 0.1],
-                                [2.6, 1.1],
-                                [2.1, 2.1],
-                                [0.6, 1.1],
-                                [1.6, 1.1]])
+    assert np.allclose(coords, [[1.1, 0.1], [2.6, 1.1],
+                                [2.1, 2.1], [0.6, 1.1], [1.6, 1.1]])
```

Comment:
yeach! I much prefer the original layout here a logially NX 2 array should look like that in teh file -- much easeir to see what values are what.